lib/forever-monitor/plugins/watch.js

```diff
@@ -21,11 +21,12 @@ exports.name = 'watch';
 //
 function watchFilter(fileName) {
```

Comment:
Thoughts on removing this function altogether and just passing the `watchIgnorePatterns` to `chokidar`? `chokidar` seems to handle the patterns much better than [watch](https://github.com/mikeal/watch) did.  We could do something like:

```javascript
  if (this.watchIgnoreDotFiles) {
    this.watchIgnorePatterns.push(/^\/?\..+/)
  }

  var opts = {
    ignoreInitial: true,
    ignored: this.watchIgnorePatterns
  };
```